### PR TITLE
chore: landing page improvements

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -168,7 +253,13 @@ const Home: React.FC = () => {
           </div>
         </div>
       </header>
+      <br></br>
       <main>
+        <br></br>
+        <br></br>
+        <center>
+          <img src="img/logos/Browsers.png" width="40%"></img>
```
Comment (mxschmitt) [inline src/pages/index.tsx:261]: nit: have lowercase file names

File: src/pages/index.tsx [hunk 2]
```diff
@@ -168,7 +253,13 @@ const Home: React.FC = () => {
           </div>
         </div>
       </header>
+      <br></br>
       <main>
+        <br></br>
```
Comment (mxschmitt) [inline src/pages/index.tsx:258]: nit: `<br/>`

**Review by mxschmitt — APPROVED** [review]

File: src/pages/index.tsx [hunk 3]
```diff
@@ -8,19 +8,107 @@ import styles from "./styles.module.css";
...
       <>
-        Capable automation for single page apps that rely on the modern web
-        platform.
+        <p>
+          <b>Browser contexts.</b> Playwright creates a browser context for each test. Browser
+          context is equivalent to a brand new browser profile. This delivers full test isolation
+          with zero overhead. Creating a new browser context only takes a handful of milliseconds.
+        </p>
+        <p>
+          <b>Log in once.</b> Save the authentication state of the context and reuse it in all the test.
```
Comment (dgozman) [inline src/pages/index.tsx:86]:
```suggestion
          <b>Log in once.</b> Save the authentication state of the context and reuse it in all tests.
```

File: src/pages/index.tsx [hunk 4]
```diff
@@ -8,19 +8,107 @@ import styles from "./styles.module.css";
...
-    title: "Test across all modern browsers",
-    description: <>Single API to automate Chromium, Firefox and WebKit.</>,
-    link: "docs/why-playwright#support-for-all-browsers",
+    title: "Any browser. Any platform. One API",
+    description: <>
+      <p>
+        <b>Cross-browser.</b> Playwright supports all modern rendering engines, including Chromium, WebKit and Firefox.
+      </p>
+      <p>
+        <b>Cross-platform.</b> Test run on Windows, Linux and macOS, locally or on CI, headless or headed.
```
Comment (dgozman) [inline src/pages/index.tsx:17]:
```suggestion
        <b>Cross-platform.</b> Test on Windows, Linux and macOS, locally or on CI, headless or headed.
```

File: src/pages/index.tsx [hunk 5]
```diff
@@ -8,19 +8,107 @@ import styles from "./styles.module.css";
...
       </>
     ),
-    link: "docs/why-playwright#powerful-automation-capabilities",
+  },
+  {
+    title: "Powerful Tooling",
+    description: (
+      <>
+        <p>
+          <b>Codegen.</b> Generate tests via performing scenarios. Save them into any language.
```
Comment (dgozman) [inline src/pages/index.tsx:97]:
```suggestion
          <b>Codegen.</b> Generate tests by recording your actions. Save them into any language.
```

**Review by dgozman — APPROVED** [review]

File: src/pages/index.tsx (hunk 1 above)
Comment (pavelfeldman) [reply src/pages/index.tsx:261]: @mxschmitt: these are the existing files: Adobe2.png  Bing.png  DHotstar.jpg  ING.png  MUI.png  Outlook.png  ReactNavigation.png  VSCode.png.

**Pull request merged**